lib/iex/lib/iex/helpers.ex

```diff
@@ -277,6 +279,106 @@ defmodule IEx.Helpers do
     end
   end
 
+  @module_info_keys [:module, :object_file, :source, :version, :compile_time, :compile_options]
+
+  @doc """
+  Prints out a list of all modules loaded and their BEAM file path.
+
+  """
+  def m() do
+    :code.all_loaded |>
+    Enum.sort |>
+    Enum.map(fn {module, path} ->m_display(module, path) end)
+
+    dont_display_result
+  end
+
+  @doc """
+  Print out available information about a loaded module.
+
+  Includes compile time, object file path, source file path,
+  version number and compile options. More complete information
+  can be found using the Erlang `module_info` function.
+  See http://www.erlang.org/doc/reference_manual/modules.html for more
+  information.
+
```

Comment:
Sorry, I don't understand why that should be cut. Shouldn't h m provide some hints?